src/golang/lib/engine/engine.go

```diff
@@ -15,7 +15,7 @@ import (
 const (
 	DefaultExecutionTimeout     = 15 * time.Minute
 	DefaultCleanupTimeout       = 2 * time.Minute
-	DefaultPollIntervalMillisec = 100
+	DefaultPollIntervalMillisec = 300
```

Comment:
Is this change intentional?

Reply:
yeah, I noticed during testing that we were polling super aggressively so I wanted to back that off a little bit.